pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -73,6 +74,9 @@ const (
 	// Service traffic, when AntreaProxy is disabled. The EndpointPod is on a remote
 	// Node and the dstPod of the reject response is on the local Node.
 	RejectNoAPServiceRemoteToLocal
+	// ErrorStatus represents that Antrea couldn't generate packetOut for current
+	// packetIn.
+	ErrorStatus
```

Comment:
nit: I would change this to something like `Unsupported` to fit in RejectType

Reply:
Sure. Will address it with other comments.

Reply:
Done.